mobile/lib/src/domain/settings/settings.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:equatable/equatable.dart';
+import 'package:polydodo/src/common/constants.dart';
+
+part 'sex.dart';
+
+class Settings extends Equatable {
+  final int age;
+  final String serverAddress;
+  final Sex sex;
+
+  Settings(
+      {this.age = 30, this.serverAddress = '0.0.0.0', this.sex = Sex.NotSet})
+      : assert(age != null || (age == null && age > 12 && age < 125)),
```

Comment:
Please use contants like MAX_AGE, MIN_AGE.

Maybe use a factory constructor so you can throw custom exceptions that you could handle in the future, in place of assert statements.